### PR TITLE
:ghost: documentation for the questionnaire yaml to be uploaded

### PR DESCRIPTION
**Pull request by shawn-hurley**
Adding documentation for users to build new assessment questionnaires

### CODE REVIEW

**Review by mansam — APPROVED** [review]
LGTM

File: docs/questionnaire-yaml.md
Comment (anarnold97) [inline docs/questionnaire-yaml.md:59]: You could use a NOTE admonition to highlight

```suggestion
> [!NOTE]
>
> 1. Anything with the word **required** must be filed out. Otherwise, the YAML will not validate on upload.
> 2. Each subsection defines a new struct/object in yaml. For instance
> ```yaml
> ...
> name: Testing
> thresholds: 
>    red: 30
>    yellow: 45
>    unknown: 5
> ...
> ```
```

Comment (shawn-hurley) [reply docs/questionnaire-yaml.md:59]: Does that note syntax work with markdown or only with asidoc?

Comment (anarnold97) [reply docs/questionnaire-yaml.md:59]: It is markdown syntax for an admonition:

image

AsciiDoc admonitions are silently different. I miss markdown so much

Comment (shawn-hurley) [reply docs/questionnaire-yaml.md:59]: Wow! never knew this will change!

Comment (shawn-hurley) [reply docs/questionnaire-yaml.md]: updated

**Comment by shawn-hurley** [comment]
@anarnold97, Thanks for the fancy new syntax to use, I will make sure that I use this everywhere now! 

@mansam, what is the process for merging? Do I need another review, or should I self-merge after your approval?

**Review by aufi — APPROVED** [review]

**Comment by mansam** [comment]
@shawn-hurley Our flow is self-merge after you get approval. If you make any significant changes after initial approval, you should generally re-request review.

**Review by mansam — APPROVED** [review]

**Pull request merged**
